tests/smoke/deno/deno.test.ts

```diff
@@ -1,3 +1,4 @@
+// @ts-ignore - need to build before this will exist
 import { decryptSops } from "../../../dist/index.js";
```

Comment:
not sure about bun, but for deno the correct solution is to import-map sops-age to ../../../src/index.ts

Reply:
we can leave this as is, more for future reference for experimentation

Reply:
For smoke tests, I wanted to use the built version vs. what's in `src/` so I can test how the various formats get bundled.  We could add a `prepare` script to build things when you install, or even put `dist` in git; but I think the above is fine for testing.